xbmc/GUIInfoManager.cpp

```diff
@@ -6911,6 +6922,13 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///     <p><hr>
 ///     @skinning_v21 **[New Infolabel]** \link ListItem_SongVideoURL `ListItem.SongVideoURL`\endlink
 ///   }
+///   \table_row3{   <b>`ListItem.BackendInstanceName`</b>,
+///                  \anchor ListItem_BackendInstanceName
+///                  _string_,
+///     @return The name used by the PVR client addon instance.
```

Comment:
You should be clear here that the value returned here is for the selected list item.

Reply:
I have changed it to `@return The name used by the PVR client addon instance for the selected item.`